openlibrary/templates/home/loans.html

```diff
@@ -0,0 +1,15 @@
+$def with(loans)
+$ carousel = loans_carousel(loans)
+$if carousel:
+    <div class="head">
+        <h2 class="inline"><a href="/account/loans">Your Loans</a></h2>
+        <p class="inline sansserif large collapse">
+          &nbsp;
+          $if len(loans):
+              You can still <a href="/borrow">borrow</a> $(5 - len(loans)) more book$("" if len(loans) == 4 else "s") until you've hit the limit!
```

Comment:
Very minor point, but having the plural check based on "4" because you subtract from 5 elsewhere is a little goofy. I'd just bite the bullet and have a clearer variable called "loans_remaining" that you can then insert and query with directly.